### PR TITLE
OPSEXP-2527: update micrometer doc & dashboards

### PR DESCRIPTION
**Pull request by alxgomz**
Ref: OPSEXP-2527

For comparison:
* newer graphs: https://rancher2.envalfresco.com/k8s/clusters/c-x949n/api/v1/namespaces/cattle-monitoring-system/services/http:rancher-monitoring-grafana:80/proxy/dashboards?query=
* older ones: https://rancher2.envalfresco.com/k8s/clusters/c-x949n/api/v1/namespaces/cattle-monitoring-system/services/http:rancher-monitoring-grafana:80/proxy/dashboards/f/T6u2dDYSk/oldalfresco?query=

### CODE REVIEW

File: docs/micrometer/README.md [hunk 1]
```diff
@@ -23,22 +33,118 @@ metrics.jvmMetricsReporter.enabled=false
...
+  jobLabel: alfresco-repo-micrometer
+  namespaceSelector:
+    any: false
+  selector:
+    matchLabels:
+      app.kubernetes.io/component: alfresco-repository
+      app.kubernetes.io/instance: acs
+```
+
+> Where the label selector and namespace should be adjusted to your own deployment.
```
Comment (gionn) [inline docs/micrometer/README.md:128]:
```suggestion
> `metadata.namespace` and `selector.matchLabels` should be adjusted to your own deployment.
```

File: docs/micrometer/README.md [hunk 2]
```diff
@@ -23,22 +33,118 @@ metrics.jvmMetricsReporter.enabled=false
...
+number of time series.
 
-### Grafana Dashboards
+This feature is enterprise only!
 
-This folder also contains dashboards that you can import in your [Grafana](https://grafana.com/) app to visualize 
-various metrics exposed by Alfresco through a Prometheus aggregation server.
+Once the Alfresco endpoint is configured to expose metrics, Prometheus must be
+configured to scrape the endpoint. The following is an example of a Prometheus
+configuration that scrapes the Alfresco endpoint:
```
[the remark this answers is not in the document]
Comment (alxgomz) [reply docs/micrometer/README.md:49]: This is what is described  line 130 when I describe the `metricRelabelings`. Is it not enough or is it unclear this relate to these regex?

File: docs/micrometer/README.md [hunk 3]
```diff
@@ -23,22 +33,118 @@ metrics.jvmMetricsReporter.enabled=false
 metrics.restMetricsReporter.enabled=false
 metrics.restMetricsReporter.path.enabled=false
 ```
-We have defined the main property metrics.enabled which defaults to false. If turned off, the web-script API will return 404.
 
-This feature is enterprise only;
+> This feature is enterprise only!
 
+We have defined the main property metrics.enabled which defaults to false. If
```
Comment (dsibilio) [inline docs/micrometer/README.md:39]:
```suggestion
We have defined the main property `metrics.enabled` which defaults to `false`. If
```

File: docs/micrometer/README.md [hunk 4]
```diff
@@ -23,22 +33,118 @@ metrics.jvmMetricsReporter.enabled=false
...
-If you want to see basic JVM metrics in a dashboard in Grafana, there are already predefined dashboards.
-Here is an example: https://grafana.com/dashboards/4701 . You could import this url directly in your Grafana.
+* [Database Metrics](/docs/micrometer/Alfresco_Enterprise_Database_Dashboard-1714473792698.json)
+* [Alfresco REST API Metrics](/docs/micrometer/Alfresco_Enterprise_REST_API_Dashboard-1714475951666.json)
+* [Alfresco JVM Metrics](/docs/micrometer/Alfresco_Enterprise_JVM_Dashboard-1714472967588.json)
+* [Alfresco Servlet Metrics](/docs/micrometer/Alfresco_Enterprise_Servlet_Dashboard-1714473338586.json)
 
+> The JVM metrics dashboard have been largely inspired from the "JVM micrometer"
+> dashboard available at https://grafana.com/dashboards/4701. One can choose to
+reuse this orignal dashboard btu be aware that Alfresco micrometer
```
Comment (dsibilio) [inline docs/micrometer/README.md:149]:
```suggestion
reuse this original dashboard but be aware that Alfresco micrometer
```

Typos

File: docs/micrometer/README.md [hunk 5]
```diff
@@ -23,22 +33,118 @@ metrics.jvmMetricsReporter.enabled=false
...
+implementation do not populate the `application` label this dashboard expects.
```
Comment (dsibilio) [inline docs/micrometer/README.md:150]:
```suggestion
implementation does not populate the `application` label this dashboard expects.
```

File: docs/micrometer/README.md [hunk 6]
```diff
@@ -23,22 +33,118 @@ metrics.jvmMetricsReporter.enabled=false
...
-This feature is enterprise only;
+> This feature is enterprise only!
 
+We have defined the main property metrics.enabled which defaults to false. If
+turned off, the web-script API will return 404.
 
-### Grafana Dashboards
+:warning: In case you want to enable REST metrics with path enabled, make sure
+your Prometheus instance can cope with the amount of time series that will be
+created, or ensure relabeling is done in scrapping configuration to reduce the
```
Comment (dsibilio) [inline docs/micrometer/README.md:44]:
```suggestion
created, or ensure relabeling is done in scraping configuration to reduce the
```

> scrapping
I assume this was meant to be `scraping` instead

File: docs/micrometer/README.md [hunk 7]
```diff
@@ -23,22 +33,118 @@ metrics.jvmMetricsReporter.enabled=false
...
+amount of time series. In the example below, `metricRelabelings` are used to
+extract the version, model and endpoint of the REST API from the `servicePath`
+label and then drop the inital label, thus avoiding to create huge amounts of
```
Comment (dsibilio) [inline docs/micrometer/README.md:47]:
```suggestion
label and then drop the inital label, thus avoiding the creation of huge numbers of
```

**Review by dsibilio — APPROVED** [review]

**Pull request merged**